Patch-Source: https://sources.debian.org/src/libde265/1.0.16-1/debian/patches/disable_tools.patch
--
Description: Disable building of some internal tools that no longer link
 because internal symbols are no longer exported.
Author: Joachim Bauch <bauch@struktur.de>
Index: libde265/Makefile.am
===================================================================
--- libde265.orig/Makefile.am
+++ libde265/Makefile.am
@@ -8,12 +8,6 @@ if ENABLE_DEC265
 SUBDIRS+=dec265
 endif
 
-if ENABLE_ENCODER
-SUBDIRS+=enc265
-endif
-
-SUBDIRS+=tools
-
 # Remove acceleration-speed test program, because it needs many internal functions (that have to be dllexported).
 # On the other hand, that tool was only used for development.
 # SUBDIRS+=acceleration-speed
